monitor/monitor_test.go

```diff
@@ -137,6 +138,51 @@ func TestNew(t *testing.T) {
 	if m.submitter.client == nil {
 		t.Errorf("Expected monitor submitter client to be non-nil")
 	}
+
+	// Creating a monitor with a DB URI and no password file should fail.
```

Comment:
Can you also add a test case that creating a monitor with a DB URI that has the wrong file system permissions fails the new check you added in e46b49d?